### PR TITLE
fuzz testing: add fpgainfo  tests

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

**Review by tswhison — DISMISSED** [review]
👍

File: tests/fuzzing/fuzz_fpgainfo.sh [hunk 1]
```diff
@@ -0,0 +1,265 @@
+#!/bin/bash
```
Comment (tswhison) [inline tests/fuzzing/fuzz_fpgainfo.sh:1]: Could you please rename this file to fuzz-fpgainfo.sh for consistency with the other scripts?

File: tests/fuzzing/main.sh [hunk 2]
```diff
@@ -28,8 +28,10 @@
 shopt -o -s nounset
 
 source fuzz-fpgaconf.sh
+source fuzz_fpgainfo.sh
```
Comment (tswhison) [inline tests/fuzzing/main.sh:31]: Please use fuzz-fpgainfo.sh as the file name for consistency.

File: tests/fuzzing/fuzz_fpgainfo.sh (hunk 1 above)
Comment (anandaravuri) [reply tests/fuzzing/fuzz_fpgainfo.sh]: rename fuzz_fpgainfo.sh to fuzz-fpgainfo.sh

File: tests/fuzzing/main.sh (hunk 2 above)
Comment (anandaravuri) [reply tests/fuzzing/main.sh]: rename fuzz_fpgainfo.sh to fuzz-fpgainfo.sh

**Review by tswhison — APPROVED** [review]

**Pull request merged**